gradle/setup.gradle

```diff
@@ -112,10 +112,28 @@ publishing {
 			artifact javadocJar
 			//consider adding extra artifacts here, conditionally on submodule's name and perhaps in an afterEvaluate block
 			afterEvaluate {
-				if (project.name == 'reactor-core') {
-					artifact rootProject.tasks.docsZip
+				// Find the docs project (it's not a java project, so we need to lookup it)
```

Comment:
```suggestion
				// Find the docs project (it's not a java project, so we need to look it up)
```

Reply:
fixed and clarified comments, se ehttps://github.com/reactor/reactor-core/commit/34c854e6ff3b2234a3bb57e858e33b24be719b36